fixed box checkered bg opacity by removing applied class from div

mveres opened the pull request:
This has the disadvantage that the checkered background div ignores other styles such as `border-radius` applied to the box - this might be an isolated corner case.

Another option would be to force `opacity: 1 !important;` and apply the showcase class as well to the checkered bg div.

I'm open to other options as well, I just didn't find a better solution.
You can see a before and after in showcase opacity class here
https://studio.backlight.dev/diff/4BeMe20hqOWTkdUL2NuJ/showcases?branch=pr:89@mveres

bashmish commented on box/src/Box.module.css:
```diff
@@ -1,11 +1,11 @@
 .checkered {
-  opacity: 1;
-  background-color: #000000;
+  opacity: 1; /* needed to avoid override by showcasing classes and styles*/
```
despite the comment, I'm not sure I understand it's still needed
I was debugging and saw it worked without it

bashmish reviewed (changes requested):
solution looks good!
but I think `overflow` is buggy and not needed, and `opacity: 1` is not needed

mveres commented:
> solution looks good! but I think `overflow` is buggy and not needed, and `opacity: 1` is not needed

I addressed these remarks, so considering it done.
Dismissing the review and merging, because this blocks me from progressing in @dockit-react.
